### ISSUE TITLE
.NET Core 3.0 support

### ISSUE THREAD

**Issue by mauroservienti** [RadicalFx/housekeeping#72]
- [x] Move `Radical` to .NET Standard https://github.com/RadicalFx/Radical/issues/258
- [x] Move `Radical.Windows` to .NET Core 3.x https://github.com/RadicalFx/Radical.Windows/pull/93
  - [x] ~~Currently blocked by~~ https://github.com/Microsoft/XamlBehaviorsWpf/pull/20 has been merged to master, scheduled to be released beginning of November. Released to nuget.
- [x] Evaluate merging `Radical.Windows` and `Radical.Windows.Presentation` into a single `Radical.Windows` package.
- ~~Move `Radical.Windows.Presentation` to .NET Core 3.x~~ merged into `Radical.Windows`
- ~~Evaluate options for DI. e.g. Move away from dedicated packages for DI containers https://github.com/RadicalFx/Radical.Windows.Presentation/pull/23. Or completely move away from DI into a pure DI direction.~~ everything moved to `IServiceCollection`
   - Rework IoC related packages

**Comment by mauroservienti** [comment]
.NET Core 3 support is now in beta. Closing this. Ongoing work is tracked by https://github.com/RadicalFx/Radical.Windows/milestone/3, https://github.com/RadicalFx/Radical/milestone/7, and https://github.com/RadicalFx/documentation/milestone/1
